apps/web/src/pages/templates/components/VariantItemCard.tsx

```diff
@@ -31,10 +32,10 @@ const FlexContainer = styled.span`
   align-items: center;
 `;
 
-const WorkflowNodeStyled = styled(WorkflowNode)<{ isError: boolean }>`
+const WorkflowNodeStyled = styled(WorkflowNode)`
+  cursor: pointer;
```

Comment:
now variant cards in the preview are clickable, meaning user can select any and preview it's template